src/opensles/OutputMixerOpenSLES.cpp

```diff
@@ -59,7 +59,7 @@ void OutputMixerOpenSL::close() {
 
     if (--mOpenCount == 0) {
         // destroy output mix object, and invalidate all associated interfaces
-        if (mOutputMixObject != NULL) {
+        if (mOutputMixObject != NULL) {  // nullptr ?
```

Comment:
Sure. Replacing NULL with nullptr anywhere is fine.

Reply:
Fixed in #578